src/server/Document/index.jsx

```diff
@@ -1,15 +1,20 @@
-import React from 'react';
-import { ChunkExtractor } from '@loadable/server';
 import path from 'path';
+
+import React from 'react';
```

Comment:
Tiny thing but why has the import order changed?

Reply:
No real reason other than consistency 1. Node modules 2. third party modules 3. local modules. 🤩